CHANGELOG.md

```diff
@@ -153,6 +153,34 @@ Multiple performance enhancements for Distributed server mailbox, IMAP, SMTP and
  - [PERF] Allow disabling host information in protocol MDC (#928)
  - JAMES-3433 MimeMessageStore StoragePolicy should be the same on read and writes
 
+## [3.7.2] - 2022-10-06
+
+### Security 
+
+ - [UPGRADE] scala-library 2.13.7 -> 2.13.9
+ - [CONF] Remove vendor URLs from smtpserver.xml default configuration examples (#1205)
+ - [Upgrade] Maven slf4j 1.7.32 -> 2.0.1
+ - [Upgrade] Maven netty4 4.1.72.Final -> 4.1.81.Final
```

Comment:
I looked a bit closer at this one and I'm a bit confused...

It refers to pretty much this commit: https://github.com/apache/james-project/commit/24b87c8bd3a320e0f01e6c2010067ff808688513

In 3.7.x we seem to have netty4 on blob-s3 and queue-activemq modules... while if I look in the pom on the root of the project, we are still at the version 3 -> 3.10.6.Final

Isn't that a problem potentially?

Reply:
No, those are on different packages -> no impact